ccm_web/Dockerfile

```diff
@@ -23,13 +23,13 @@ FROM node:14-slim AS prod
 ENV NODE_ENV=production
 WORKDIR /app
 
-COPY --from=base /base/package.json ./
+COPY --from=base /base/package.json /base/start.sh ./
```

Comment:
Wonder if we should be also adding `package-lock.json` (i.e. `package*.json`)...

Reply:
Done [`bfa4929` (#125)](https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/125/commits/bfa4929fc93af2504b3d05abe82d631dc6a602f4)